packages/core/scripts/gas-rebate/rebates/Rebate_1.json

```diff
@@ -1,50 +1,152 @@
 {
   "rebate": 1,
-  "fromBlock": 10764504,
-  "toBlock": 10784145,
+  "fromBlock": 10752294,
```

Comment:
@claytonroche here's a sample output that would be the real UMA rebates for the votes:
- Admin 10
- Admin 11
- USDETH-Sept